.circleci/config.yml

```diff
@@ -23,15 +23,15 @@ jobs:
             cd $UNDERLAY_WS && vcs import src < ros2_dependencies.repos
             DEBIAN_FRONTEND=noninteractive  rosdep install -y --ignore-src \
               --skip-keys "pybind11_vendor" --skip-keys "ompl" --skip-keys "slam_toolbox" --from-paths src
-            MAKEFLAGS="-j4 -l1" colcon build --symlink-install --cmake-args --parallel-workers 1 \
-              --executor sequential --packages-up-to pcl_ros octomap_msgs nav2_costmap_2d
+            MAKEFLAGS="-j4 -l1" colcon build --symlink-install --executor sequential --packages-up-to nav2_costmap_2d
             source $UNDERLAY_WS/install/setup.bash
-            cd $OVERLAY_WS && rosdep install -y --ignore-src --from-paths src --skip-keys "octomap"
+            cd $OVERLAY_WS && DEBIAN_FRONTEND=noninteractive rosdep install -y --ignore-src --from-paths src
       - run:
           name: Debug Build
           command: |
             source $UNDERLAY_WS/install/setup.bash
-            cd $OVERLAY_WS && colcon build --parallel-workers 1
+            cd $OVERLAY_WS && MAKEFLAGS="-j4 -l1" colcon build --executor sequential
```

Comment:
What's the value of this change?

Reply:
It's to reduce the ram usage for circle ci.